src/contributing/style_guide.md

```diff
@@ -0,0 +1,85 @@
+# Style Guide
+
+These are some general style guidelines for articles which appear on `docs.telemetry.mozilla.org` (DTMO). Reading this will help you write material that is more useful to others.
```

Comment:
These are some general style guidelines available for articles that appear on `docs.telemetry.mozilla.org` (DTMO). Reading these style guidelines can help you write content that your audience can easily understand.